library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -29,23 +29,25 @@
 
 import rx.Observable;
 import rx.functions.Func1;
+import rx.subjects.PublishSubject;
 
 public class ListViewCard extends FrameLayout
 {
     private static final String TAG = "ListViewCard";
 
     private List<Item> mItems = new ArrayList<>();
 
+    private final PublishSubject<Item> mItemClicks = PublishSubject.create();
+    private final PublishSubject<Item> mIconClicks = PublishSubject.create();
+
     private Observable<Void> mButtonClicks;
```

Comment:
Any reason you converted this back to a plain Observable as opposed to a PublishSubject? 

Using a PublishSubject allows subscribing at any point and seems to be good practice for surfacing view bindings as observables.